lib/mwoffliner.lib.js

```diff
@@ -895,26 +384,26 @@ function execute(argv) {
           logger.log(`Css dependencies of ${articleId} : ${styleDependenciesList}`);
 
           const allDependenciesWithType = [
-            { type: 'js', moduleList: jsDependenciesList },
-            { type: 'css', moduleList: styleDependenciesList }
+            { type: "js", moduleList: jsDependenciesList },
+            { type: "css", moduleList: styleDependenciesList }
           ];
 
           allDependenciesWithType.forEach(({ type, moduleList }) =>
             moduleList.forEach(oneModule => downloadAndSaveModule(oneModule, type))
           );
 
-          // Saving, as a js module, the jsconfigvars that are set in the header of a wikipedia page
+          // Saving, as a js module, the jsconfiglets that are set in the header of a wikipedia page
```

Comment:
Please please fix all these incorrect find-replace changes. I am just flagging a few.

Reply:
@darkenvy Do you still plan to fix the last few small things reported by @subbuss or do you disagree with them? Should I do it?